src/style.css

```diff
@@ -0,0 +1,108 @@
+/* #region global */
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Avenir", sans-serif;
+  font-size: 16px;
+  font-weight: normal;
+  color: #3d4e61;
+
+  --background-color: #fff;
+  --border: 1px solid #e1e7ed;
+  --box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  --border-radius: 4px;
+}
+
+body {
+  margin-top: 0;
+}
+
+/* #endregion */
+
+/* #region containers */
+.serch_bar-container-small,
+.serch_bar-container {
+  position: relative;
+  margin-top: 20px;
+}
+
+.serch_bar-container {
+  height: 70px;
+}
+
+.serch_bar-container-small {
+  height: 42px;
+}
+
+/* #endregion  */
+
+/* #region inputs and logos */
+.input_big,
```

Comment:
create common class for inputs and move all repetitive styles there

Reply:
fixed

Reply:
fixed